src/types.ts

```diff
@@ -56,6 +56,16 @@ export const ARRAY_CONSTRUCTORS = {
 export interface ColorizeFeature {
   idsToFeatureValue: DataTexture;
   featureValueToColor: DataTexture;
+  /**
+   * A mapping from the current frame number to the ID offset for that frame.
+   * This is used for data where the raw IDs are not globally-unique. Adding the
+   * offset to the raw ID gives the unique, global ID that can be used to index
+   * into the `idsToFeatureValue` and other colorize-related data textures.
+   *
+   * For each raw ID `i` at some time `t`, the global ID is `i +
+   * timeToIdOffset[t]`.
+   */
```

Comment:
so in the case of globally unique ids, we just have to use a zero offset?  it might be worth saying that you have to do it explicitly by passing zeros in...